### PR TITLE
Remove Icon usage from Hug, Select, and Step

### PR DESCRIPTION
**Pull request by unscsprt**
Removed `<Icon>` from:

- Hug
- Step
- Select

### CODE REVIEW

**Comment by unscsprt** [comment]
[Tree-shaking](https://trello.com/c/n6r9l61r/3-tree-shaking)

File: packages/core/src/Hug.js [hunk 1]
```diff
@@ -24,28 +22,25 @@ const BorderConcealer = styled(Box)`
   }
 `
 
-const HideableIcon = styled(Icon)`
-  ${display};
-`
+const Hug = ({ bg, color, p, fontSize, icon, iconDisplay, ...props }) => {
```
Comment (James300) [inline packages/core/src/Hug.js]: What could iconDisplay be?

Comment (unscsprt) [reply packages/core/src/Hug.js]: Apparently that's a string for the CSS display property

Comment (James300) [reply packages/core/src/Hug.js]: I feel like it's overkill to have that much power without doing something with css but I guess it's an existing thing 🤷‍♂

**Pull request merged**